upgrades-docker-migrate.adoc

```diff
@@ -0,0 +1,57 @@
+= Migrate to new docker settings
+:toc: left
+:toclevels: 1
+:numbered:
+:source-highlighter: pygments
+:pygments-style: friendly
+
+While `taiga-docker` has been a pretty successful way for many people to have a Taiga self-hosted, the Taiga team receives a bunch of questions about the docker configuration. We have released a new `taiga-docker` version with the main settings based in an `.env` file. The main goal behind this change is to make it way easier to have a basi Taiga up and running.
+
+For those of you who already have a self-hosted Taiga running with docker, this tutorial will guide you through the steps of the migration.
+
+[[taigadckr-to-taigadckr]]
+== Summary
+
+If you are familiar with docker and docker compose, this is the sensible summary:
+- save the modifications made in the `docker-compose.yml` and `docker-compose-inits.yml` files
+- update the repository `taiga-docker`
+- add the configuration values to the `.env` file
+- add the customisation values to the `docker-compose.yml` file
+- launch all services again
```

Comment:
The list it's not being rendered correctly. You missed a new line before the list itself.

![image](https://user-images.githubusercontent.com/4539017/220947993-2d025b75-c9a1-46d1-9242-b6df25ffaa0f.png)

Reply:
fixed!